docs/image_assertions.rst

```diff
@@ -0,0 +1,117 @@
+Image Assertions
```

Comment:
Is this a standard term?  I would have expected it to be something like "Screenshot Comparisons".